app/autotranslate/server/deeplTranslate.js

```diff
@@ -151,6 +151,9 @@ class DeeplAutoTranslate extends AutoTranslate {
 						.join('\n');
 					translations[language] = this.deTokenize(Object.assign({}, message, { msg: translatedText }));
 				}
+				if (result.statusCode === 456) {
+					SystemLogger.error('Error translating message', result.message);
+				}
 			} catch (e) {
 				SystemLogger.error('Error translating message', e);
```

Comment:
@sampaiodiego do you think this should throw a Meteor error (so the user knows why it failed)?